### ISSUE TITLE
Cannot login on Windows 10 + Chrome

### ISSUE THREAD

**Issue by goxr3plus** [flickr-downloadr/flickr-downloadr-gtk#111]
![image](https://user-images.githubusercontent.com/20374208/72969125-9116c500-3dcd-11ea-87de-7374709f28e7.png)

**Comment by ucsendre** [comment]
Hello, Same error happened when I tried to authorize the application. It isreally simple and easy to solve the problem: you have to change the protocol ( **https**:// to **http**:// ) in the browser address bar while leaving the other parts of the callback URL intact. Just delete the "s" and press enter.

**Comment by goxr3plus** [comment]
Done but now i get :

![image](https://user-images.githubusercontent.com/20374208/73830441-5bca9800-480d-11ea-8ed4-f88ce5afef83.png)

**Comment by ucsendre** [comment]
:(
I use the same version and it worked without any issues.

**Comment by goxr3plus** [comment]
@ucsendre This one :) ? https://github.com/flickr-downloadr/flickr-downloadr-gtk/releases/tag/v3.2.3.1

**Comment by ucsendre** [comment]
I mean this one:
https://flickrdownloadr.com/installer/windows/flickrdownloadr-3.2.3.1-windows-installer.exe
:D

**Comment by goxr3plus** [comment]
@ucsendre Yes not working :( aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa Windows 10 + Chrome

**Comment by goxr3plus** [comment]
@cutedaffodil Bro can you help :) ?

**Comment by ucsendre** [comment]
I will try to reproduce the error.

**Comment by ucsendre** [comment]
Did you use the default settings after successful login?
It works fine for me on this PC too.
![image](https://user-images.githubusercontent.com/22783189/73844075-1fecfe00-4820-11ea-9251-4736f80070cf.png)

**Comment by ucsendre** [comment]
Do you have any third party safety software (firewall, malware or virus scanner) installed on your PC?
Try uninstall and re-install.
Open the browser and log in to your flickr account before starting installation.
The app will open a new browser tab for the authorization, but you will be logged in on that tab too.
Delete the "s" from https on the callback page as above, hit enter and go back to the app.
Click login, select default settings and it should be fine.

**Comment by cutedaffodil** [comment]
@goxr3plus - there is something going on at your end causing this issue - like @ucsendre we can't see it at our end.

@ucsendre - thanks for helping out with the replacing of `https` with `http` - looks like something changed at flickr's end - the app is suggesting to redirect back to an `http://localhost...` URL, but after login they're sending the users to an `https://localhost...` one

**Comment by Dot-Dash** [comment]
A similar problem occurred when authenticating on Firefox on Ubuntu. Firefox provided a more specific message which may help: SSL_ERROR_RX_RECORD_TOO_LONG.  I confirm that the workaround of changing https to http worked here also. 
Versions: flickr downloadr beta v3.3.4.1, Firefox v. 86.0, Ubuntu 18.04.5 LTS 64-bit
![Firefox SSL Error](https://user-images.githubusercontent.com/6632885/111652263-53e57180-87dd-11eb-9584-18367d9f4504.jpg)
